plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/conversions/NullabilityConversion.kt

```diff
@@ -21,6 +19,14 @@ class NullabilityConversion(context: NewJ2kConverterContext) : RecursiveConversi
         if (element is JKTypeCastExpression) {
             element.updateNullability()
         }
+        if (element is JKMethod) {
```

Comment:
`element` can't be both `JKTypeCastExpression` and `JKMethod`. Better to use "else if" or "when".